src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.ts

```diff
@@ -496,6 +499,10 @@ export class ProgrammingExerciseUpdateComponent implements AfterViewInit, OnDest
                 title: 'artemisApp.programmingExercise.wizardMode.detailedSteps.generalInfoStepTitle',
                 valid: this.exerciseInfoComponent?.formValid ?? false,
             },
+            {
```

Comment:
Needed for the validation bar
![image](https://github.com/user-attachments/assets/823f5d8c-24b1-4b6a-a6d5-67c559103a9e)